src/components/NewOrderAmortization.vue

```diff
@@ -16,6 +16,10 @@
       </a>
     </div>
     <div class="modal-body">
+      <div v-if="order.missMatchedPayments.length > 0" class="text-danger">
+        <i class="fa fa-exclamation-circle text-danger" aria-hidden="true"></i>
+        The account name doesn't match with that on credit report   
```

Comment:
Let's use `The debit account does not correspond to the account recorded on the bank statement.`